Library/Homebrew/cleanup.rb

```diff
@@ -36,10 +42,18 @@ def self.cleanup_logs
       end
     end
 
-    def self.cleanup_cellar
-      Formula.installed.each do |formula|
-        cleanup_formula formula
+    def self.cleanup_cellar(formulae = Formula.installed)
+      unremovable_kegs = []
+      formulae.each do |formula|
+        cleanup_formula formula, unremovable_kegs
```

Comment:
I agree, I wasn't sure whether we needed to maintain the previous behavior of failing when cleanup_formula fails for individual calls, so I did it this way. I'll change this to use an instance variable.